src/main/java/cs361/battleships/models/Board.java

```diff
@@ -114,6 +114,15 @@ public Result sonarPulse(int x, char y) {
 		return finalResult;
 	}
 
+	public Result moveShips(char moveDir) {
+		for (Ship s : ships) {
```

Comment:
Good job breaking down the moveShips functionality into its own function instead of trying to attach it to another process